DependencyInjection/Compiler/DescriptionEnhancerPass.php

```diff
@@ -59,11 +70,14 @@ public function process(ContainerBuilder $container)
             ));
         }
 
-        $inactiveEnhancers = array_diff($enhancerNames, $enabledEnhancers);
-        foreach ($inactiveEnhancers as $inactiveEnhancer) {
-            $container->removeDefinition((string) $enhancers[$inactiveEnhancer]);
-            unset($enhancers[$inactiveEnhancer]);
-        }
+        // sort enhancers, higher = more priority
+        usort($enhancers, function ($a, $b) {
+            return -strcmp($a[0], $b[0]);
```

Comment:
what about strcmp($b[0], $a[0])? the `-` in front of strcmp confused me for a bit...